images/linux/scripts/installers/python.sh

```diff
@@ -10,47 +10,9 @@ source $HELPER_SCRIPTS/document.sh
 # Install Python, Python 3, pip, pip3
 apt-get install -y --no-install-recommends python python-dev python-pip python3 python3-dev python3-pip
```

Comment:
Was planning on removing all these extra downloaded versions of Python but I'll leave this for another PR (want to make sure PyPy + tools cache is working correctly for Ubuntu)